tests/Makefile

```diff
@@ -22,7 +22,64 @@ start-test-stack: venom-postgres.cid venom-mysql.cid venom-redis.cid venom-imap.
 stop-test-stack:
 	@for f in `ls -1 *.cid`; do docker stop `cat $${f}`; docker rm `cat $${f}`; done; rm -f *.cid
 
-run-test: 
+run-test:
 	@(cd ../cli/venom; go build && ./venom run --stop-on-failure  --log debug ../../tests/*.yml)
 
 
+TMP_DIR = /tmp/ovh/venom
```

Comment:
can you rework all those impacts to integrate the kafka test stack as all the other stacks.

the goal is to provide a single make command `start-test-stack` and them being able to run all the tests with a command `venom run *.yml` - from our CI server or on laptops

Reply:
Hi, I am reworking my test setup to adhere to the existing commands. I will need to change the kafka image for 2 main reasons: 1. https://github.com/spotify/docker-kafka is archived and 4 years old, so I don't have hope to find help for it; 2. I need a kafka local cluster with schema-registry

Reply:
no problem if we need to change the kafka image if the basic integration tests are passed.